test/test_checkFile.js

```diff
@@ -26,8 +26,8 @@ describe('koa-validate' , function(){
 				return __dirname+"/temp";
 			})).delete();
 			require('fs').unlinkSync(__dirname+'/temp');
-			require('fs').unlinkSync(__dirname+'/'+require('path').basename(this.request.body.files.file.path));
-			require('fs').unlinkSync(__dirname+'/tempdir/'+require('path').basename(this.request.body.files.file.path));
+			require('fs').unlinkSync(__dirname+'/'+require('path').basename(this.request.files.file.path));
+			require('fs').unlinkSync(__dirname+'/tempdir/'+require('path').basename(this.request.files.file.path));
```

Comment:
One more breaking change spot:
Newer versions of koa-body don't support request.body.files any longer.

https://github.com/koajs/koa-body/pull/89